docs: fix the commands described in the README (#328)
maxGraph/maxgraph: README.md

- this is the version used by GitHub Actions
- nvm users can run `nvm use`. If the Node version is not installed, nvm will state how to install the required version.

Note: maxGraph relies on npm workspaces to build.

In the project root directory, execute

```sh
$ npm install
```

To watch the core package, execute:

```sh

> $ nnpm run dev -w packages/core

Author: ```suggestion
$ npm run dev -w packages/core
```

```

> To run the html (vanilla-js) version of [Storybook](https://storybook.js.org/), execute (in the project root directory)

Author: ```suggestion
To run the examples provided as [Storybook](https://storybook.js.org/) stories, execute:
```
